docs/getting-started/metrics.adoc

```diff
@@ -2,18 +2,74 @@
 
 Mobile application Metrics SDK
 
-== Adding dependency
+== Using metrics
 
-Add dependency to your application module
-----
-dependencies {
-    implementation group: 'org.aerogear.android', name: 'metrics'
-}
-----
+Metrics is part of the Core SDK and as such no additional dependency is required.
+If you want to make use of Metrics all you have to do is to make sure that your `mobile-services.json` contains a configuration section for it.
```

Comment:
Without configuration metrics will still be enabled but it will not send them to server (just log to the console)
This will give developers chance to see our application metrics if they have some remote logging solution already and do not with to use metrics backend.